src/components/accordion/Accordion.styles.ts

```diff
@@ -4,8 +4,16 @@ import {
   commonHoverShadow
 } from '~/styles/app-theme/custom-shadows'
 
+const noIconTitleStyles = {
+  fontSize: { md: '20px', sm: '13px' },
```

Comment:
you can use h6 variant for md and i guess caption or body2 for sm

Reply:
changed